dotnet/src/Connectors/Connectors.OpenAI/Core/OpenAIStreamingChatMessageContent.cs

```diff
@@ -33,17 +34,37 @@ internal OpenAIStreamingChatMessageContent(
         string modelId,
         IReadOnlyDictionary<string, object?>? metadata = null)
         : base(
-            chatUpdate.Role.HasValue ? new AuthorRole(chatUpdate.Role.Value.ToString()) : null,
+            null,
             null,
             chatUpdate,
             choiceIndex,
             modelId,
             Encoding.UTF8,
             metadata)
     {
-        this.ToolCallUpdates = chatUpdate.ToolCallUpdates;
-        this.FinishReason = chatUpdate.FinishReason;
-        this.Items = CreateContentItems(chatUpdate.ContentUpdate);
+        try
+        {
+            this.FinishReason = chatUpdate.FinishReason;
+
+            if (chatUpdate.Role.HasValue)
```

Comment:
When the happens for one this also happens for the other two properties below due to the same bug root cause.

When an exception happens, those properties don't need to be set.

```suggestion
            if (chatUpdate.Role.HasValue)
```

Reply:
@RogerBarret0, please a comment to remove the try-catch block later when the issue is fixed, so we don't forget to remove it.

Reply:
And/or create an issue in our backlog and add to the next sprint